mkosi/__init__.py

```diff
@@ -2806,6 +2796,104 @@ def make_disk(
     return make_image(context, msg=msg, skip=skip, split=split, tabs=tabs, root=context.root, definitions=definitions)
 
 
+def make_oci(context: Context, root_layer: Path, dst: Path) -> None:
+    ca_store = dst / "blobs" / "sha256"
+    ca_store.mkdir(parents=True)
+
+    layer_diff_digest = hash_file(root_layer)
+    maybe_compress(
+        context,
+        context.config.compress_output,
+        context.staging / "rootfs.layer",
+        context.staging / "rootfs.layer",
+    )
+    layer_digest = hash_file(root_layer)
+    root_layer.rename(ca_store / layer_digest)
+
+    creation_time = (
+        datetime.datetime.fromtimestamp(
+            context.config.source_date_epoch, tz=datetime.timezone.utc
+        )
+        if context.config.source_date_epoch
+        else datetime.datetime.now(tz=datetime.timezone.utc)
+    ).isoformat()
+
+    oci_config = {
+        "created": creation_time,
+        "architecture": context.config.architecture.to_oci(),
+        # Name of the operating system which the image is built to run on.
+        # SHOULD use value listed in the golang documentation for $GOOS.
+        # https://github.com/opencontainers/image-spec/blob/v1.0.2/config.md#properties
```

Comment:
I was confused by this comment. "SHOULD" makes it sound like something is not implemented here. Maybe just say "Name of the operating system which the image is built to run on as defined by https://github.com/opencontainers/image-spec/blob/v1.0.2/config.md#properties."